src/ripple/beast/core/SystemStats.cpp

```diff
@@ -61,24 +60,29 @@ getStackBacktrace()
         std::distance(std::begin(stack), std::end(stack)),
         stack, nullptr);
 
-    HeapBlock<SYMBOL_INFO> symbol;
-    symbol.calloc (sizeof (SYMBOL_INFO) + 256, 1);
-    symbol->MaxNameLen = 255;
-    symbol->SizeOfStruct = sizeof (SYMBOL_INFO);
+    // Allow symbols that are up to 1024 bytes long
+    std::size_t const nameLength = 1024;
+    std::size_t const nameSize = nameLength * sizeof(((SYMBOL_INFO *)0)->Name);
```

Comment:
This is old pre-C++11 code destined for the bit bucket. I wouldn't stare at it too closely.